Error/ErrorHandler.php

```diff
@@ -129,4 +133,32 @@ public function handleErrors(ExecutionResult $executionResult, $throwRawExceptio
             $executionResult->extensions['warnings'] = array_map(['GraphQL\Error', 'formatError'], $exceptions['extensions']['warnings']);
         }
     }
+
+    /**
+     * Tries to convert a raw exception into a user warning or error
+     * that is displayed to the user.
+     *
+     * @param \Exception $rawException
+     * @return \Exception
+     */
+    protected function convertException(\Exception $rawException = null)
+    {
+        if (empty($rawException)) {
+            return $rawException;
+        }
+
+        $types = [
```

Comment:
Same here, you can added to the service arg [here](https://github.com/adri/GraphQLBundle/blob/4308b8c3cd1a367a7ffdda56f3fddfba222877a9/DependencyInjection/OverblogGraphQLExtension.php#L109)...